README.md

```diff
@@ -80,40 +91,17 @@ You can then run `make build-charon-ml` to build the ML library, or even simply
 `make` to build the whole project (Rust and OCaml). Finally, you can run the
```

Comment:
Above, should we instead suggest running `opam install . --deps-only` to install opam packages? It might be more future-proof when adding packages for charon-ml

Reply:
If that works, that sounds much better. Is there a reason we don't already suggest that?

Reply:
Is the opam package/install tested on CI? If so, I think we should switch.

Reply:
It's not: https://github.com/AeneasVerif/charon/issues/185

Reply:
I'll do it in the next PR

Reply:
https://github.com/AeneasVerif/charon/pull/594